error_tracker/django/migrations/0002_auto_20201018_1311.py

```diff
@@ -6,7 +6,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('error_tracker', '0001_initial'),
+        ('django', '0001_initial'),
```

Comment:
do we still need this?

Reply:
removed

Reply:
will need to add a mention for updating on README for makemigrations on existing installation.